packages/@aws-cdk/aws-stepfunctions/lib/state-machine.ts

```diff
@@ -334,11 +426,41 @@ export interface IStateMachine extends IResource {
    */
   readonly stateMachineArn: string;
 
+  /**
+   * The ARN of the executions
+   * @attribute
+   */
+  readonly executionArn: string;
+
```

Comment:
the `@attribute` tag corresponds to attributes of the cloudformation resource
i.e. the state machine name is an attribute. you can check this out from the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-stepfunctions-statemachine.html#aws-resource-stepfunctions-statemachine-return-values) where the `@attribute` corresponds to the `Fn::GetAtt` entries

Reply:
updated